### ISSUE TITLE
coqide 8.10 should depend on gnome-icon-theme (especially on Mac)

### ISSUE THREAD

**Issue by Blaisorblade** [coq/opam-coq-archive#945]
After `opam install coqide` on my Mac (from https://github.com/ocaml/opam-repository/pull/15017, which follows this repo), I had to install `gnome-icon-theme` for the toolbar to work. That package isn't always installed, and not just on Mac. Seems we'd need to depend on a new `conf-gnome-icon-theme` package containing:

```
depexts: [
  ["gnome-icon-theme"] {os = "macos" & os-distribution = "homebrew"}
]
```
and other such lines. I wrote that following https://github.com/ocaml/opam-repository/blob/master/packages/conf-gnomecanvas/conf-gnomecanvas.2/opam.

**Comment by Blaisorblade** [comment]
And this fix is also needed on opam (see #944); nevertheless, it should be done here to apply to future releases and to dev packages.

**Comment by Blaisorblade** [comment]
Closed by #960 and ocaml/opam-repository#15050.
